wikiconv/ingest_revisions/BUILD

```diff
@@ -1,5 +1,27 @@
 py_binary(
     name = "dataflow_main",
     srcs = ["dataflow_main.py"],
-    deps = ["//ingest_utils:wikipedia_revisions_ingester"],
+    deps = ["//wikiconv/ingest_revisions/ingest_utils:wikipedia_revisions_ingester"],
+)
+
+# Requires cloud project auth,
```

Comment:
Is the command literally `cloud project auth`? If not, could you give the command needed?

Reply:
It's documented here
https://github.com/conversationai/wikidetox/tree/master/wikiconv#setup-the-environment

I'm working on making this into a proper test with mocks, that seems easier.'
Added a TODO.